lib/ansible/modules/cloud/cloudstack/cs_instance_facts.py

```diff
@@ -227,6 +232,20 @@ def get_instance(self):
                         break
         return self.instance
 
+    def get_volumes(self, instance):
+        volume_details = []
+        if instance:
+            args                = {}
+            args['account']     = instance['account'] if 'account' in instance.keys() else None
+            args['domainid']    = instance['domainid'] if 'domainid' in instance.keys() else None
+            args['projectid']   = instance['projectid'] if 'projectid' in instance.keys() else None
```

Comment:
this can be simplified: 
~~~python
args['account'] = instance.get('account')
args['domainid'] = instance.get('domainid')
args['projectid'] = instance.get('projectid')
~~~